src/nupic/engine/TestNode.cpp

```diff
@@ -116,7 +117,9 @@ namespace nupic
 
 
   TestNode::TestNode(AnyPointer::Reader& proto, Region* region) :
-    RegionImpl(region)
+    RegionImpl(region),
+    computeCallback_(nullptr)
```

Comment:
The missing `computeCallback_(nullptr)` was resulting in sporadic SEGFAULT when running the new link serialization test.